test/functional/logos_feature_bip341_sighash.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Logos Foundation
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the BIP341 sighash."""
```

Comment:
Can you put some more details here.  E.g. `Test that we can broadcast transactions via .... using new BIP341 sighashes`